spec/rubocop/cop/style/negated_if_else_condition_spec.rb

```diff
@@ -50,6 +50,21 @@
     RUBY
   end
 
+  it 'registers an offens and corrects a multiline ternary' do
+    expect_offense(<<~RUBY)
+      !x ?
+      ^^^^ Invert the negated condition and swap the ternary branches.
+        do_something :
+        do_something_else # comment
+    RUBY
+
+    expect_correction(<<~RUBY)
+      x ?
+        do_something_else :
+        do_something # comment
+    RUBY
+  end
```

Comment:
This doesn't actually correct the comment properly but I wanted to add a test for this case just to document behaviour. Really no one should be writing multiline ternaries like this. 😏